confluent_kafka/src/confluent_kafka.c

```diff
@@ -405,11 +409,13 @@ static PyMethodDef Message_methods[] = {
 	  "\n"
 	},
 	{ "set_value", (PyCFunction)Message_set_value, METH_O,
+	  "  Set the field 'Message.value' with new value.\n"
```

Comment:
Document param and type, see an example here:
https://github.com/confluentinc/confluent-kafka-python/blob/master/confluent_kafka/src/confluent_kafka.c#L741